dev.khcrysalis.PlumeImpactor.json

```
@@ -0,0 +1,92 @@
+{
+    "app-id": "dev.khcrysalis.PlumeImpactor",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "25.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable",
+        "org.freedesktop.Sdk.Extension.llvm21"
+    ],
+    "command": "plumeimpactor",
+    "finish-args": [
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--share=ipc",
+        "--device=all",
+        "--talk-name=org.freedesktop.Notifications",
+        "--filesystem=/run/usbmuxd"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm21/bin",
+        "env": {
+            "CARGO_HOME": "/run/build/plumeimpactor/cargo",
+            "CARGO_NET_OFFLINE": "true",
+            "BINDGEN_EXTRA_CLANG_ARGS_x86_64-unknown-linux-gnu": "--sysroot=/usr -isystem /usr/include -isystem /usr/include/x86_64-linux-gnu -isystem /usr/lib/sdk/llvm21/lib/clang/21/include",
+            "BINDGEN_EXTRA_CLANG_ARGS_x86_64_unknown_linux_gnu": "--sysroot=/usr -isystem /usr/include -isystem /usr/include/x86_64-linux-gnu -isystem /usr/lib/sdk/llvm21/lib/clang/21/include",
+            "BINDGEN_EXTRA_CLANG_ARGS_aarch64-unknown-linux-gnu": "--sysroot=/usr -isystem /usr/include -isystem /usr/include/aarch64-linux-gnu -isystem /usr/lib/sdk/llvm21/lib/clang/21/include",
+            "BINDGEN_EXTRA_CLANG_ARGS_aarch64_unknown_linux_gnu": "--sysroot=/usr -isystem /usr/include -isystem /usr/include/aarch64-linux-gnu -isystem /usr/lib/sdk/llvm21/lib/clang/21/include"
+        }
+    },
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "libzip",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DBUILD_SHARED_LIBS=ON",
+                "-DBUILD_TOOLS=OFF",
+                "-DBUILD_REGRESS=OFF",
+                "-DBUILD_EXAMPLES=OFF",
+                "-DENABLE_BZIP2=ON",
+                "-DENABLE_LZMA=ON",
+                "-DENABLE_ZSTD=ON"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://libzip.org/download/libzip-1.11.4.tar.xz",
+                    "sha256": "8a247f57d1e3e6f6d11413b12a6f28a9d388de110adc0ec608d893180ed7097b"
+                }
+            ]
+        },
+        {
+            "name": "plumeimpactor",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 ./wxwidgets-extract.py",
+                "cargo --offline fetch --manifest-path Cargo.toml",
+                "bash -c 'export RUSTFLAGS=\"-C link-arg=-L/app/lib -C link-arg=-lnotify\"; cargo --offline build --release -p plumeimpactor'",
```

```suggestion
                "RUSTFLAGS=\"-C link-arg=-L/app/lib -C link-arg=-lnotify\" cargo --offline build --release -p plumeimpactor",
```

this should work. no need to overcomplicate.